mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,36 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
+    """Store projects editorial info at the time of publish"""
+
+    label = "Collect Project Edit Info"
+    order = pyblish.api.CollectorOrder
+    hosts = ["maya"]
+    families = [
+        # More will come
+        "mindbender.animation"
+    ]
+
+    def process(self, context):
+        import os
+
+        mindbender_fps = os.getenv("MINDBENDER_FPS")
```

Comment:
Considering all environment variables are prefixed `MINDBENDER_`, maybe you can try and think of a way to collect *all* of them in one fell swoop, in some elegant way?